src/UI/Component/Symbol/Glyph/Factory.php

```diff
@@ -951,4 +951,33 @@ public function time($action = null);
      * @return \ILIAS\UI\Component\Symbol\Glyph\Glyph
      */
     public function close($action = null);
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *      The More Glyph is used to symbolize a set of actions that can not
+     *      be displayed directly due to scarce space.
+     *   composition: >
+     *      The More Glyph uses the glyphicon-option-vertical.
+     *   effect: >
+     *       Clicking the More Glyph displays the actions hidden behind the Glyph.
+     *
+     * context:
+     *    - This Glyph is currently used in the responsive view of the Meta Bar.
+     *    - This Glyph is currently used in the responsive view of the Main Bar.
+     *
```

Comment:
replace "list" with "set of entries":

- The Mini-Action-Drop-Down in ListGUI without text is used to offer a set of actions that cannot be displayed directly due to scarce space. This is different because the set of entries of the More Glyph does not entail actions.
- The Show-More /Show Less Button in Timeline unhides a full individual entry of a timeline. Entries are caped at a certain length and Show-More-Buttons allow viewing all the content of this entry. This is different, because the unhidden entirety is an individual entry and not a set of entries.
- The Show-More /Show Less Button in filtered Categories with loads of objects shows the next x objects in the list GUI. This is different, because what is shown is not an entirety but a part of an entirety.